README.md

```diff
@@ -1,11 +1,12 @@
 # GOV.UK Smoke Tests
 
-Automated tests that describe high level user journeys which touch multiple
-applications within the GOV.UK stack.
+A suite of [Cucumber](https://cucumber.io/) tests that probe GOV.UK frontend and backend publishing functionality. The tests [use Selenium to manipulate a headless Chrome browser](features/support/env.rb).
 
-The master branch of the tests is frequently run in all environments, triggered by deployments of most GOV.UK applications, CDNs and associated dependencies ([check here](https://github.com/alphagov/govuk-puppet/search?l=HTML%2BERB&q=smokey)).
+The tests are run in two different ways:
 
-The tests also run in [a continuous Smokey loop](https://github.com/alphagov/govuk-puppet/blob/master/modules/monitoring/templates/smokey-loop.conf). We use the Smokey loop to provide Icinga alerts for major features.
+- On demand, to check if a change breaks something. This is done using the [Smokey job in Jenkins](https://github.com/alphagov/govuk-puppet/blob/b103dd3b4adcc8c39343dd85b68f4f5b93e38d9d/modules/govuk_jenkins/manifests/jobs/smokey.pp) when deploying GOV.UK applications, CDN config and associated dependencies ([check here](https://github.com/alphagov/govuk-puppet/search?l=HTML%2BERB&q=smokey)).
```

Comment:
This link doesn't seem to have any value IMO.

```suggestion
- On demand, to check if a change breaks something. This is done using the [Smokey job in Jenkins](https://github.com/alphagov/govuk-puppet/blob/b103dd3b4adcc8c39343dd85b68f4f5b93e38d9d/modules/govuk_jenkins/manifests/jobs/smokey.pp) when deploying GOV.UK applications, CDN config and associated dependencies.
```

Reply:
Fair point. I've rewritten with specific examples in https://github.com/alphagov/smokey/pull/927/commits/57cab57b5d968fef48446f88007cfa7d6c443af0.